components/compliance-service/scanner/db.go

```diff
@@ -217,28 +216,31 @@ func (s *Scanner) UpdateResult(ctx context.Context, job *types.InspecJob, output
 	if job.NodeStatus == types.StatusFailed || job.NodeStatus == types.StatusAborted {
 		if inspecErr != nil {
 			result.Result = inspecErr.Message
+		} else {
+			result.Result = fmt.Sprintf("unable to complete scan on node. please check the logs for more information. node id ref: %s", job.NodeID)
```

Comment:
i'm wondering if maybe the lack of else logic here contributed to https://github.com/chef/automate/issues/2252
i'm hoping this else logic + more detailed msgs in the logs will help